frontend/src/components/core/Grid.tsx

```diff
@@ -0,0 +1,18 @@
+import styled from 'styled-components';
```

Comment:
I didn't see this file and its contents used anywhere - is this going to be incorporated someplace in the future, or no? If not, maybe remove this file?

Reply:
Just deleted it, nice catch - this was a remnant of my initial attempt at the dashboard two column layout